gradle.properties

```diff
@@ -13,6 +13,9 @@ researchgateReleaseVersion=2.8.0
 slf4jVersion=1.7.30
 ballerinaGradlePluginVersion=1.0.0
 gsonVersion=2.8.8
+lz4Version=1.3.0
+marshallingVersion=2.0.5.Final
+protobufVersion=2.6.1
```

Comment:
```suggestion
protobufVersion=3.20.3
```

Lets update `protobuf` version since there is a [security vulnerability with this version](https://github.com/ballerina-platform/module-ballerina-tcp/actions/runs/4022661299/jobs/6912636478#step:7:36)

Reply:
Fixed with [a955c52](https://github.com/ballerina-platform/module-ballerina-tcp/pull/879/commits/a955c52763ae46bde1587f8aacae101dbebd8600)